lionweb/src/main/kotlin/com/strumenta/kolasu/lionweb/LionWebLanguageConverter.kt

```diff
@@ -272,6 +273,17 @@ class LionWebLanguageConverter {
             val newEnumeration = Enumeration(lionwebLanguage, kClass.simpleName)
             newEnumeration.id = (lionwebLanguage.id ?: "unknown_language") + "_" + newEnumeration.name
             newEnumeration.key = newEnumeration.name
+
+            val entries = kClass.java.enumConstants
+            entries.forEach { entry ->
+                newEnumeration.addLiteral(
+                    EnumerationLiteral(newEnumeration, entry.name).apply {
+                        setID(newEnumeration.id + "-" + entry.name)
+                        setKey(id)
```

Comment:
They key should be unique within the language, so it could make sense not to include the language name in it.
Also, we should remove invalid characters, if present